java/app/src/main/java/com/google/android/gms/samples/wallet/activity/CheckoutActivity.java

```diff
@@ -100,6 +115,8 @@ protected void onCreate(Bundle savedInstanceState) {
     // It's recommended to create the PaymentsClient object inside of the onCreate method.
     paymentsClient = PaymentsUtil.createPaymentsClient(this);
     possiblyShowGooglePayButton();
+
+    requestPaymentCardOcrIntent();
```

Comment:
I'd suggest using a consistent nomenclature with other similar methods (eg.: `possiblyShowCardOcrRecognitionButton`)

Reply:
Updated. Thanks for the suggestion.